File: Python/assignment13100.py
def validateEmail(email):
    return '@' in email and '.' in email
def sanitizeInput(message):
    return message.replace('&','')
email = input("Please enter an email.")
message = input("Please enter a message")
checkedEmail = validateEmail(email)
sanitizedInput = sanitizeInput(message)
print(f"Original email: ", email)
print(f"Original message: ", message)
print(f"Is email valid? ", checkedEmail)
print(f"Sanitized input ", sanitizedInput)

File: C++/Assignment13100.cpp
#include <iostream>
#include <stdlib.h>
#include <string>
using namespace std;
bool validateEmail(string email){
    return email.contains('@') && email.contains('.');
}
string sanitizeUserInput(string message){
    return message.replace()
}
int main(){
    string email;
    string message;
    cout << "Please enter an email" << endl;
    cin >> email;
    cout << "Please enter a message" << endl;
    cin >> message;
    return 0;
}